### PR TITLE
Updated README to give it better flow and mention the other ways to use SigMF

### PR DESCRIPTION
**Pull request by 777arc**
You can view the render of the new README here https://github.com/777arc/SigMF

It's still missing an index of SigMF examples.

Sorry commit isn't signed, I wanted to use github web gui to preview changes as I made them

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -24,9 +24,39 @@ sharing of data, prevents the "bitrot" of datasets wherein details of the
...
+            "core:frequency": 915000000,
+        }
+    ],
+    "annotations": []
+}
+```
+
+## Using SigMF
+
+There are currently four ways that you can use SigMF:
```
Comment (gmabey) [inline README.md:53]: suggest "currently" -> "at least", or maybe just "four community-supported projects for using SigMF".

Comment (777arc) [reply README.md]: Great suggestion, I've made that change

File: README.md [hunk 2]
```diff
@@ -24,9 +24,39 @@ sharing of data, prevents the "bitrot" of datasets wherein details of the
...
+            "core:frequency": 915000000,
+        }
+    ],
+    "annotations": []
+}
+```
+
+## Using SigMF
+
+There are four community-supported projects for using SigMF:
```
Comment (gmabey) [inline README.md:53]: Hmm -- so actually, there are currently just 3 community-supported projects.  Maybe replace "four" with "several" and move "4." to a new paragraph that explains that it's *so* easy to parse the file yourself, so "manually" is a really good option too.

File: README.md [hunk 3]
```diff
@@ -153,20 +187,20 @@ meta.tofile('example.sigmf-meta') # extension is optional
...
 
-#### Is this specific to wireless communications?
+### Is this specific to wireless communications?
 
 *No*, similar to the response, above, the goal is to create something that is
 generally applicable to _signal processing_, regardless of whether or not the
 application is communications related.
 
-#### It seems like some issues take a long time to resolve?
+### It seems like some issues take a long time to resolve?
```
Comment (gmabey) [inline README.md:203]: I agree that the `###` is more appropriate and looks better.

File: README.md (hunk 2 above)
Comment (777arc) [reply README.md]: what do you think of the change i just made

File: README.md (hunk 3 above)
Comment (777arc) [reply README.md:203]: Yeah I imagined having a TOC one day, want subsections to make sense

File: README.md (hunk 2 above)
Comment (gmabey) [reply README.md]: I think it's great.

File: README.md [hunk 4]
```diff
@@ -24,9 +24,39 @@ sharing of data, prevents the "bitrot" of datasets wherein details of the
...
+    ],
+    "annotations": []
+}
+```
+
+## Using SigMF
+
+There are at least four ways you can use SigMF today, thanks to the community-supported projects:
+
+1. Within **Python**, using the Python package included in this repo and discussed below (avoid installing with pip for the time being, 1.0.0 isn't on PyPI yet)
```
Comment (777arc) [inline README.md:55]: @bhilburn what do you think about this note?  Do you also want a banner at the top of the readme?

Comment (gmabey) [reply README.md:55]: I suggest that the note about `pip` be relocated closer to line 83.  Also, aside from the timing of sigmf arriving on PyPI, a `.deb` is a 90% solution, which is pretty easily generated using this command, and could be included with the release files:
```bash
fpm -s python -t deb --python-bin /usr/bin/python3 --python-package-name-prefix python3 setup.py
```

File: README.md [hunk 5]
```diff
@@ -24,9 +24,39 @@ sharing of data, prevents the "bitrot" of datasets wherein details of the
...
 
-(Taken from the
-[Introduction](https://github.com/gnuradio/SigMF/blob/master/sigmf-spec.md#introduction)
-of the specification document.)
+SigMF signal recordings typically involve a data file (e.g. a binary file of IQ
+or RF samples) and a metadata file containing plain text that describes the data.
+Together these files represent one recording, such as `example.sigmf-data` and
+`example.sigmf-meta`.  Here is a minimal example of a SigMF `.sigmf-meta` file:
+
+```yaml
```
Comment (bhilburn) [inline README.md:32]: Why mark this is yaml instead of json?

File: README.md [hunk 6]
```diff
@@ -24,9 +24,39 @@ sharing of data, prevents the "bitrot" of datasets wherein details of the
 capture are lost over time, and makes it possible for different tools to operate
 on the same dataset, thus enabling data portability between tools and workflows.
 
-(Taken from the
-[Introduction](https://github.com/gnuradio/SigMF/blob/master/sigmf-spec.md#introduction)
-of the specification document.)
+SigMF signal recordings typically involve a data file (e.g. a binary file of IQ
```
Comment (bhilburn) [inline README.md:27]: nitpicky, but should be a comma at the end - `e.g.,`

File: README.md (hunk 3 above)
Comment (bhilburn) [reply README.md:203]: 👍🏼

File: README.md (hunk 4 above)
Comment (bhilburn) [reply README.md:55]: Hold on, let me see about the pip thing

**Review by bhilburn — CHANGES_REQUESTED** [review]

File: README.md (hunk 4 above)
Comment (bhilburn) [reply README.md:55]: Okay, pip is now up-to-date with v1.0.0 👍🏼

**Review by jacobagilbert — COMMENTED** [review]
@777arc Ben has some good fixes that should be resolved but after that, I approve.

**Comment by 777arc** [comment]
Done!

**Review by jacobagilbert — APPROVED** [review]

**Comment by jacobagilbert** [comment]
Thanks @777arc this was a good addition. Merged.

**Pull request merged**